ChefActions/Chopping/ChoppingChefAction.cs

```diff
@@ -1,8 +1,14 @@
+using Godot;
+using GColl = Godot.Collections;
 using System.Collections.Generic;
 using System.Linq;
 
 public partial class ChoppingChefAction : ChefAction
 {
+
+    [Export]
+    public GColl.Array<Ingredient> MustBeCooked { get; set; }
+
```

Comment:
I remember helping get this to work but it must have never gotten checked in. Not major